client/blocks/image-editor/index.jsx

```diff
@@ -35,7 +35,7 @@ const ImageEditor = React.createClass( {
 
 	propTypes: {
 		// Component props
-		media: PropTypes.object,
```

Comment:
Could we relax this, I'm getting 
```
Failed prop type: Required prop `media` was not specified in `ImageEditor`.
```
When editing in the image editor modal on the /post page